### PR TITLE
Update German translation for 1.8.9

### PR DESCRIPTION
**Pull request by wlhlm**
Ping @Vexatos.

### CODE REVIEW

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 1]
```diff
@@ -1,181 +1,256 @@
...
+ritual.BloodMagic.testRitual=Testritual
+ritual.BloodMagic.waterRitual=Ritual des vollen Frühlings
+ritual.BloodMagic.lavaRitual=Serenade des Nethers
+ritual.BloodMagic.greenGroveRitual=Ritual des grünen Hains
+ritual.BloodMagic.jumpRitual=Ritual des hohen Sprungs
+ritual.BloodMagic.wellOfSufferingRitual=Quelle des Leidens
+ritual.BloodMagic.featheredKnifeRitual=Ritual der gefederten Klinge
+ritual.BloodMagic.regenerationRitual=Ritual der Regeneration
+
+# Chat
```
Comment (TehNut) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: What about the chat messages?

Comment (wlhlm) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Oops, forgot that :+1:

**Comment by wlhlm** [comment]
Updated with missing translations.

**Comment by TehNut** [comment]
Looks good, but I'd like somebody to verify before I merge this

**Comment by wlhlm** [comment]
That's why I've mentioned @Vexatos.

File: src/main/resources/assets/bloodmagic/lang/en_US.lang [hunk 2]
```diff
@@ -39,11 +39,6 @@ item.BloodMagic.slate.imbued.name=Imbued Slate
 item.BloodMagic.slate.demonic.name=Demonic Slate
 item.BloodMagic.slate.ethereal.name=Ethereal Slate
 
-item.BloodMagic.focus.weak.name=Teleposition Focus
-item.BloodMagic.focus.enhanced.name=Enhanced Teleposition Focus
-item.BloodMagic.focus.reinforced.name=Reinforced Teleposition Focus
-item.BloodMagic.focus.demonic.name=Demonic Teleposition Focus
-
 item.BloodMagic.orb.weak.name=Weak Blood Orb
```
Comment (WayofTime) [inline src/main/resources/assets/bloodmagic/lang/en_US.lang]: ...Why did you delete these?

Comment (TehNut) [inline src/main/resources/assets/bloodmagic/lang/en_US.lang]: These are duplicates.

Set 1: https://github.com/WayofTime/BloodMagic/blob/1.8-Rewrite/src/main/resources/assets/bloodmagic/lang/en_US.lang#L31-L34

Set 2: https://github.com/WayofTime/BloodMagic/blob/1.8-Rewrite/src/main/resources/assets/bloodmagic/lang/en_US.lang#L42-L45

Comment (WayofTime) [inline src/main/resources/assets/bloodmagic/lang/en_US.lang]: Oh ok

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 3]
```diff
@@ -1,181 +1,256 @@
...
+
+tooltip.BloodMagic.activated=Aktiviert
+tooltip.BloodMagic.deactivated=Deaktiviert
+
+tooltip.BloodMagic.sigil.air.desc=&oIch fühl mich schon leichter...
+tooltip.BloodMagic.sigil.bloodLight.desc=&oIch sehe ein Licht!
+tooltip.BloodMagic.sigil.compression.desc=&oDiamanthände
+tooltip.BloodMagic.sigil.divination.desc=&oSpähe in die Seele
+tooltip.BloodMagic.sigil.divination.currentAltarTier=Ausbaustufe: %d
+tooltip.BloodMagic.sigil.divination.currentEssence=Gespicherte Lebensessenz: %d LP
```
Comment (Aphobis) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Typo. Should be: "Gespeicherte Lebensessenz".

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 4]
```diff
@@ -1,181 +1,256 @@
...
+tooltip.BloodMagic.sigil.divination.currentAltarCapacity=Maximaler Kapazität: %d LP
```
Comment (Aphobis) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Should be: "Maximale Kapazität:"

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 5]
```diff
@@ -1,181 +1,256 @@
...
+item.BloodMagic.slate.imbued.name=Erfüllte Tafel
+item.BloodMagic.slate.demonic.name=Dämonische Tafel
+item.BloodMagic.slate.ethereal.name=Ätherische Tafel
+
+item.BloodMagic.orb.weak.name=Schwacher Blutorb
+item.BloodMagic.orb.apprentice.name=Lehrlingsblutorb
+item.BloodMagic.orb.magician.name=Blutorb des Magiers
+item.BloodMagic.orb.master.name=Blutorb des Meisters
+item.BloodMagic.orb.archmage.name=Blutorb des Erzmagiers
+item.BloodMagic.orb.transcendent.name=Überragendes Blutorb
```
Comment (BluSunrize) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Blutorb is a male noun, so "Überragender".
Also, I'd actually consider calling it "Transzendenter Blutorb", because it sounds so much more awesome. x3

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 6]
```diff
@@ -1,181 +1,256 @@
...
+tooltip.BloodMagic.sigil.divination.currentEssence=Gespicherte Lebensessenz: %d LP
+tooltip.BloodMagic.sigil.divination.currentAltarCapacity=Maximaler Kapazität: %d LP
+tooltip.BloodMagic.sigil.water.desc=&oHat jemand nach unendlichem Wasser gefragt?
+tooltip.BloodMagic.sigil.lava.desc=&oHEIẞ! NICHT ESSEN!
+tooltip.BloodMagic.sigil.void.desc=&oBesser als Swiffer®!
+tooltip.BloodMagic.sigil.greenGrove.desc=&oUmweltverträglich
+tooltip.BloodMagic.sigil.magnetism.desc=&oIch habe eine anziehende Persönlichkeit
+tooltip.BloodMagic.sigil.suppression.desc=&oBesser als Telekinese
+tooltip.BloodMagic.sigil.haste.desc=&o42 Dosen Koffein später...
+tooltip.BloodMagic.sigil.fastMiner.desc=&oHacke wei ein Weltmeister
```
Comment (pennywis3) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Typo: "wei" should be "wie"

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 7]
```diff
@@ -1,181 +1,256 @@
...
+item.BloodMagic.bound.pickaxe.name=Gebundene Spitzhacke
+item.BloodMagic.bound.axe.name=Gebundene Axt
+item.BloodMagic.bound.shovel.name=Gebundene Schaufel
+
+item.BloodMagic.bucket.lifeEssence.name=Lebenseimer
+
+item.BloodMagic.scribe.water.name=Elementar-Gravurwerkzeug: Wasser
+item.BloodMagic.scribe.fire.name=Elementar-Gravurwerkzeug: Feuer
+item.BloodMagic.scribe.earth.name=Elementar-Gravurwerkzeug: Erde
+item.BloodMagic.scribe.air.name=Elementar-Gravurwerkzeug: Leuft
```
Comment (abecderic) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Should be: "Elementar-Gravurwerkzeug: Luft"
air = Luft, c.f. line 132

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang (hunk 1 above)
Comment (Galaxy2Alex) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Line 218 and 219 are missing a "=".

Comment (wlhlm) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Fixed in 15dd0d1bda84b3adad217d5bcf1a20fe14832c9b

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang (hunk 5 above)
Comment (wlhlm) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: :+1:

**Comment by wlhlm** [comment]
Thanks for the feedback and @copyyy for sending in a PR.

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 8]
```diff
@@ -1,181 +1,256 @@
...
+tooltip.BloodMagic.sigil.seer.currentEssence=Gespeicherte Lebensessenz: %d LP
+tooltip.BloodMagic.sigil.seer.currentAltarCapacity=Maximaler Kapazität: %d LP
+tooltip.BloodMagic.sigil.seer.currentCharge=Ladung: %d
+tooltip.BloodMagic.sigil.phantomBridge.desc=&oLaufen auf dünner Luft...
+tooltip.BloodMagic.sigil.whirlwind.desc=&oBesser keinen Rock tragen
+tooltip.BloodMagic.sigil.enderSeverance.desc=&oBringt Enderman in Notlage
+
+tooltip.BloodMagic.bound.sword.desc=&oErlegen der Schwachen
+tooltip.BloodMagic.bound.pickaxe.desc=&oZerstören von Stein ohne Erbarmen
+tooltip.BloodMagic.bound.axe.desc=&oDemonische Entwälderung
```
Comment (pennywis3) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: "Dämonische Entwaldung" would be the correct translation. Personally, I'd go with "Dämonsicher Kahlschlag" which, IMO, sounds less dry.

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang [hunk 9]
```diff
@@ -1,181 +1,256 @@
...
+tooltip.BloodMagic.sigil.haste.desc=&o42 Dosen Koffein später...
+tooltip.BloodMagic.sigil.fastMiner.desc=&oHacke wei ein Weltmeister
+tooltip.BloodMagic.sigil.elementalAffinity.desc=&oAchtung! Fallende Feuerfische
+tooltip.BloodMagic.sigil.seer.desc=&oWenn allsehend nicht genug ist
+tooltip.BloodMagic.sigil.seer.currentAltarProgress=Fortschritt: %d LP/ %s LP
+tooltip.BloodMagic.sigil.seer.currentAltarProgress.percent=Fortschritt: %s
+tooltip.BloodMagic.sigil.seer.currentAltarConsumptionRate=Verbrauch: %d LP
+tooltip.BloodMagic.sigil.seer.currentAltarTier=Ausbaustufe: %d
+tooltip.BloodMagic.sigil.seer.currentEssence=Gespeicherte Lebensessenz: %d LP
+tooltip.BloodMagic.sigil.seer.currentAltarCapacity=Maximaler Kapazität: %d LP
```
Comment (pennywis3) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: This should read "Maximale Kapazität: %d LP"

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang (hunk 8 above)
Comment (wlhlm) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: The latter definitely sounds better. Will update.

**Comment by Simon04090** [comment]
I'm on mobile right now, so I can only take a look at this tomorrow.

**Comment by wlhlm** [comment]
Sure. No rush :)

File: src/main/resources/assets/bloodmagic/lang/de_DE.lang (hunk 1 above)
Comment (BonsaiXXL) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: The ritual of the full spring is currently translated as a season, not a water spring. It would be "Ritual der vollen Quelle" instead

Comment (wlhlm) [inline src/main/resources/assets/bloodmagic/lang/de_DE.lang]: Makes sense. Will Fix :+1:

**Comment by Vexatos** [comment]
Looks decent to me :+1:

**Comment by TehNut** [comment]
Squash that last commit down and I'll merge

**Comment by wlhlm** [comment]
Sure. I originally left it in, because I wanted to give credit.

**Comment by wlhlm** [comment]
Rebased and ready.
